includes/Customizer.php

```diff
@@ -0,0 +1,589 @@
+	 */
+	private function get_view_type_choices( $view_type ) {
+
+		if ( empty( $view_type ) ) {
+			return [ 'circles' => __( 'Circles', 'web-stories' ) ];
+		}
+
+		$view_type = array_keys( $view_type );
+
+		$view_type_choices = [];
+
+		if ( in_array( 'circles', $view_type ) ) {
+			$view_type_choices['circles'] = __( 'Circles', 'web-stories' );
+		}
+
+		if ( in_array( 'grid', $view_type ) ) {
+			$view_type_choices['grid'] = __( 'Grid', 'web-stories' );
+		}
+
+		if ( in_array( 'list', $view_type ) ) {
+			$view_type_choices['list'] = __( 'List', 'web-stories' );
+		}
+
+		if ( in_array( 'carousel', $view_type ) ) {
+			$view_type_choices['carousel'] = __( 'Carousel', 'web-stories' );
+		}
+
+		return $view_type_choices;
+
+	}
+
+
+	/**
+	 * Gets the order choices.
+	 *
+	 * @param array $order An array of order support.
+	 *
+	 * @return array An array of order choices.
+	 */
+	private function get_order_choices( $order ) {
+
+		$order_choices = [];
+
+		if ( empty( $order ) ) {
+			return [
+				'latest'               => __( 'Latest', 'web-stories' ),
+				'oldest'               => __( 'Oldest', 'web-stories' ),
+				'alphabetical'         => __( 'A -> Z', 'web-stories' ),
+				'reverse-alphabetical' => __( 'Z -> A', 'web-stories' ),
+			];
+		}
+
+		$order = array_keys( $order );
+
+		if ( in_array( 'latest', $order ) ) {
+			$order_choices['latest'] = __( 'Latest', 'web-stories' );
+		}
+
+		if ( in_array( 'oldest', $order ) ) {
+			$order_choices['oldest'] = __( 'Oldest', 'web-stories' );
+		}
+
+		if ( in_array( 'alphabetical', $order ) ) {
+			$order_choices['alphabetical'] = __( 'A -> Z', 'web-stories' );
+		}
+
+		if ( in_array( 'reverse-alphabetical', $order ) ) {
+			$order_choices['reverse-alphabetical'] = __( 'Z -> A', 'web-stories' );
+		}
+
+		return $order_choices;
+
+	}
+
+	/**
+	 * Checks whether the given option is enabled or not.
+	 *
+	 * @param string $option_name The name of the option to check.
+	 *
+	 * @return boolean Returns true if the given option is enabled otherwise false.
+	 */
+	private function is_option_enabled( $option_name ) {
+		$setting = $this->wp_customize->get_setting( "story-options[{$option_name}]" );
+		return ( $setting instanceof \WP_Customize_Setting && true === $setting->value() );
+	}
+
+	/**
+	 * Verifies the current view type.
+	 *
+	 * @param string $view_type View type to check.
+	 *
+	 * @return bool Whether or not current view type matches the one passed.
+	 */
+	private function is_view_type( $view_type ) {
+		$setting = $this->wp_customize->get_setting( 'story-options[view_type]' );
+		return ( $setting instanceof \WP_Customize_Setting && $view_type === $setting->value() );
+	}
+
+	/**
+	 * Validates the number of story setting value.
+	 *
+	 * @param \WP_Error $validity WP_Error object.
+	 * @param int       $value    Value to be validated.
+	 *
+	 * @return \WP_Error
+	 */
+	public function validate_number_of_stories( $validity, $value ) {
+		$value = intval( $value );
```

Comment:
It is better to just cast these? 
```suggestion
		$value = (int) $value;
```